src/main/java/com/marklogic/kafka/connect/source/TimestampKeyGenerator.java

```diff
@@ -0,0 +1,16 @@
+package com.marklogic.kafka.connect.source;
+
+import java.time.Instant;
+
+public class TimestampKeyGenerator implements KeyGenerator {
```

Comment:
Per the story - "rowNumber is simply the index of the row in the result set" - I think the way this should work is:

1. After a PlanInvoker runs a plan, it should instantiate a KeyGenerator (KG)
2. It invokes the KG for each row
3. Thus, this class - TKG - should have a rowNumber field that defaults to 1, and it gets incremented after each call to getUniqueKey